spec/unit/hanami/settings_spec.rb

```diff
@@ -43,6 +43,30 @@
     end
   end
 
+  describe "#inspect" do
+    it "hides values" do
+      config_class = Class.new(described_class) do
```

Comment:
Minor nitpick, this is actually a `settings_class`

Reply:
Updated.